server/src/storage/compressed_data_manager.rs

```diff
@@ -28,13 +28,16 @@ use tracing::info;
 use tracing::{debug, debug_span};
 
 use crate::errors::ModelarDbError;
+use crate::storage::data_transfer::DataTransfer;
 use crate::storage::time_series::CompressedTimeSeries;
 use crate::types::{CompressedSchema, Timestamp};
 use crate::StorageEngine;
 
 /// Stores data points compressed as models in memory to batch compressed data before saving it to
 /// Apache Parquet files.
-pub(super) struct CompressedDataManager {
+pub struct CompressedDataManager {
```

Comment:
Could this be `pub(super)` if a method is added to `StorageEngine`?

Reply:
Fixed.